network-api/networkapi/buyersguide/tests.py

```diff
@@ -72,53 +87,27 @@ def test_product_view_404(self):
         """
         Test that the product view raises an Http404 if the product name doesn't exist
         """
-        request = self.factory.get('/en/privacynotincluded/products/this is not a product')
-        self.assertRaises(Http404, product_view, request, 'this is not a product')
+        response = self.client.get('/en/privacynotincluded/products/this is not a product')
+        self.assertEqual(response.status_code, 404, 'this is not a product')
 
     def test_category_view_404(self):
         """
         Test that the category view raises an Http404 if the category name doesn't exist
         """
-        request = self.factory.get('/en/privacynotincluded/categories/this is not a category')
-        self.assertRaises(Http404, category_view, request, 'this is not a category')
+        response = self.client.get('/en/privacynotincluded/categories/this is not a category')
+        self.assertEqual(response.status_code, 404, 'this is not a category')
 
     def test_category_view(self):
         """
-        Test that the category view returns a 200 for both slug and name URLs
+        Test that the category view returns a 302 for /privacynotincluded/ to /en/privacynotinclded/
+        and return a 404 for a slug that doesn't exist
         """
         response = self.client.get('/privacynotincluded/categories/Smart%20Home/')
-        self.assertEqual(response.status_code, 302, 'The category "Smart Home" should work by name')
+        self.assertEqual(response.status_code, 404, 'Smart%20Home is an invalid slug and should return a 404.')
 
         response = self.client.get('/privacynotincluded/categories/smart-home/')
         self.assertEqual(response.status_code, 302, 'The category "Smart Home" should work by slug')
 
-    def test_drive_by_clear_cache(self):
-        """
-        regular users should not be able to trigger clear_cache
-        """
-        authenticated = self.user.is_authenticated
-
-        self.client.logout()
-        response = self.client.post('/api/buyersguide/clear-cache/')
-        self.assertEqual(response.status_code, 403, 'standard user is not permitted to clear BG cache')
-
-        if authenticated is True:
-            self.client.force_login(self.user)
-
-    def test_authenticated_clear_cache(self):
-        """
-        authenticated users can trigger clear_cache
-        """
-        authenticated = self.user.is_authenticated
-
-        self.client.force_login(self.user)
-        response = self.client.post('/api/buyersguide/clear-cache/')
-        self.assertEqual(response.status_code, 302, 'authenticated user is permitted to clear BG cache')
-        self.assertEqual(response.url, '/cms/', 'clearing sends users to product page')
-
-        if authenticated is False:
-            self.client.logout()
```

Comment:
Tests removed because they were for the old clear_cache function for the buyersguide and product pages. This is no longer used, so it got removed.